lib/app/modules/alarmRing/views/alarm_ring_view.dart

```diff
@@ -76,6 +77,12 @@ class AlarmControlView extends GetView<AlarmControlController> {
                                       controller.currentlyRingingAlarm.value,
                                 );
                               } else {
+                                if (controller.currentlyRingingAlarm.value
+                                        .deleteAfterGoesOff ==
+                                    true) {
+                                  IsarDb.deleteAlarm(controller
```

Comment:
This would work for Isar, maybe we should do the same for firestore as well - if the alarm's are shared, then it can be deleted. Let's show the option to auto-delete only to the owner of the alarm (it should be stored in firestore, should be easy to implement)